.travis.yml

```diff
@@ -12,8 +11,6 @@ matrix:
     env: TOX_ENV=py27-oauth2client3
   - python: "2.7"
     env: TOX_ENV=py27-oauth2client4
-  - python: "3.4"
```

Comment:
Did you by chance try apitools with Python 3.6, 3.7?

Reply:
I just had a quick go, but it failed so I think that can be dealt with separately to this:

https://travis-ci.org/hugovk/apitools/builds/530685178